### ISSUE TITLE
Unable to combine Mutations in schema

### ISSUE THREAD

**Issue by robinelvin** [graphql-python/graphene#501]
Combining query classes works but combining mutation classes does not appear to work.

Based on @yfilali's comment on #408 -

```
class Query(App1Query, App2Query, graphene.ObjectType):
    pass

class Mutations(App1Mutations, App2Mutations, graphene.ObjectType):
    pass


schema = graphene.Schema(query=Query, mutation=Mutations)

```

Server responds with:

```Mutations fields must be a mapping (dict / OrderedDict) with field names as keys or a function which returns such a mapping.```

Both

```
schema = graphene.Schema(query=Query, mutation=App1Mutations)
```
and
```
schema = graphene.Schema(query=Query, mutation=App2Mutations)
```
work as expected.

**Note:** I am not trying to nest the mutations.

**Comment by robinelvin** [comment]
Solved.

Ok, it was my mistake. I hadn't made the Mutations derive from ```graphene.AbstractType``` however, the error message that came back is very misleading.

**Comment by syrusakbary** [comment]
This will be solved in `2.0` as you can use normal inheritance without using `AbstractType` :)

**Comment by jkimbo** [comment]
Looks like this issue has been solved. Closing.
